projects/ngrx.io/content/guide/effects/lifecycle.md

```diff
@@ -167,32 +167,31 @@ By default, effects are merged and subscribed to the store. Implement the `OnRun
 Usage:
 
 <code-example header="user.effects.ts">
+import { Observable } from 'rxjs';
+import { exhaustMap, takeUntil, tap } from 'rxjs/operators';
 import { Injectable } from '@angular/core';
 import {
   Actions,
-  Effect,
   OnRunEffects,
   EffectNotification,
   ofType,
+  createEffect,
 } from '@ngrx/effects';
-import { Action } from '@ngrx/store';
-import { Observable } from 'rxjs';
-import { exhaustMap, takeUntil, tap } from 'rxjs/operators';
 
 @Injectable()
 export class UserEffects implements OnRunEffects {
   constructor(private actions$: Actions) {}
 
   updateUser$ = createEffect(() =>
-    this.actions$.pipe(
-      ofType('UPDATE_USER'),
-      tap(action => {
-        console.log(action);
-      })
-    ),
-  { dispatch: false });
-
-  ngrxOnRunEffects(resolvedEffects$: Observable&lt;EffectNotification&gt;) {
+      this.actions$.pipe(
+        ofType('UPDATE_USER'),
+        tap(action => {
+          console.log(action);
+        })
+      ),
+    { dispatch: false });
+
+  ngrxOnRunEffects(resolvedEffects$: Observable<EffectNotification>) {
```

Comment:
We want to use the HTML entities here, `Observable&lt;EffectNotification&gt;`.
Otherwise the `<` and `>` brackets won't show up

Reply:
Ooh, I didn't realise that. Hmm I wonder if this can be improved somewhat as it makes maintenance really difficult. I suspect a better solution would be to make the `CodeComponent` require a markdown preformatted block so that the source could be an actual typescript file. I'll fix this up, but raise a different issue to fix this properly.

Reply:
Hold off on creating an issue for now please, we (Brandon) is currently looking for alternatives for our docs.

Reply:
Oh right I already raised https://github.com/ngrx/platform/issues/2338 at least from that issue I will find out what the plan is ;)